lomond/parser.py

```diff
@@ -18,8 +18,13 @@ class ParseEOF(ParseError):
     """End of Stream."""
 
 
+class ParseOverflow(Exception):
```

Comment:
This new exception is thrown if data is supplied in `feed` when the parser has finished. This wasn't ever occurring, but at least its implemented now. It could catch issues in the future.